assets/email/new-login.txt

```diff
@@ -0,0 +1,7 @@
+Hi there!
+
+This is just an email to warn you that there was a new successful login to your account in Padloc, from {{ location }}.
+
+If this was you, there's no action necessary, otherwise you might want to make sure your trusted devices haven't been compromised, or to remove potentially compromised devices from your trusted list inside the app.
```

Comment:
You might also want to add a note about the device in question having been removed from the list of trusted devices.